Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -510,8 +510,13 @@ extension TelemetryWrapper {
         case (.action, .tap, .goToSettingsDefaultBrowserOnboarding, _, _):
             GleanMetrics.DefaultBrowserOnboarding.goToSettingsPressed.add()
         // Onboarding
-        case (.action, .press, .dismissedOnboarding, let slideNum, _):
-            GleanMetrics.Onboarding.finish.record(extra: [.slideNum : slideNum])
+        case (.action, .press, .dismissedOnboarding, _, let extras):
```

Comment:
AFAICT previously `slideNum` would always be `nil` because it was matching against the 4th argument, which is `EventValue` rather than `extras`.